ecosystem/sep-0038.md

```diff
@@ -130,8 +130,8 @@ Name | Type | Description
 Name | Type | Description
 -----|------|------------
 `asset` | string | The [Asset Identification Format](#asset-identification-format) value.
-`sell_delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to sell/deliver funds to the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response.
-`buy_delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to buy/retrieve funds from the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response.
+`sell_delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to sell/deliver funds to the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response. If the delivery method is not necessary for providing accurate quotes and expirations, the server can omit this attribute.
+`buy_delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to buy/retrieve funds from the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response. If the delivery method is not necessary for providing accurate quotes and expirations, the server can omit this attribute.
```

Comment:
💡  I still think it's valuable to require the `/info` endpoint to display all available delivery methods, even if there's only one option.

Reply:
Hmm thats a good point, it seems there are two reasons why delivery methods should be listed:
- To show what delivery methods are supported / available
- To provide the client with the `name`'s so they can be used in future SEP-38 requests

I think we should keep the current text because the 1st reason can be satisfied in the transaction flow in which the quote is used. For example, I could omit the delivery methods from SEP-38's `GET /info` but I could still display the delivery options in the SEP-24 interactive flow. And both SEP-6 and SEP-31 have a [fields](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#fields) object in their own `GET /info` responses that could be used to document the possible delivery methods as well.

Reply:
Could SEP-38 be used independently from the other SEPs? SEP-38 doesn't mention anything about SEPs 6, 24, or 31 and I believe it's better that SEPs contain all the information needed within themselves, rather than rely on other SEPs. WDYT?

Reply:
> Could SEP-38 be used independently from the other SEPs? 

It could, although if the purpose of SEP-38 is to provide quotes to be used in transactions, and delivery method info isn't needed to calculate quotes, I think listing supported delivery methods should be the responsibility of the application facilitating the transaction.